example/tutorial/bookstore/README.md

```diff
@@ -0,0 +1,467 @@
+
+
+### Admin
+
+If the bookstore app is not yet running, start it by running `fresh` in the bookstore directory:
+
+    go/src/github.com/qor/qor/example/tutorial/bookstore/01 [bookstore (master)] $ fresh
+
+Go to [http://localhost:9000/admin](http://localhost:9000/admin) and you should see the main admin interface:
+
+![qor_admin](https://raw.githubusercontent.com/qor/qor/docs_and_tutorial/example/tutorial/bookstore/screenshots/qor_admin1.png)
+
+The menu at the top gets created by adding your models as resources to the admin in [main.go](https://github.com/qor/qor/blob/docs_and_tutorial/example/tutorial/bookstore/01/main.go):
+
+	Admin := admin.New(&qor.Config{DB: &db})
+
+	Admin.AddResource(
+		&User{},
+		&admin.Config{
+			Menu: []string{"User Management"},
+			Name: "Users",
+		},
+	)
+
+you can see how the rest of the resources was added in [resources.go](https://github.com/qor/qor/blob/master/example/tutorial/bookstore/01/app/resources.go), the `db` object referenced here is set up in [models.go](https://github.com/qor/qor/blob/master/example/tutorial/bookstore/01/app/models/models.go#L66:L72)
+
+Go ahead and go to the authors admin and add an author...
+
+![qor_admin_add_author](https://raw.githubusercontent.com/qor/qor/docs_and_tutorial/example/tutorial/bookstore/screenshots/qor_admin_add_author.png)
+
+... and then a book via the admin:
+
+![qor_admin_add_book](https://raw.githubusercontent.com/qor/qor/docs_and_tutorial/example/tutorial/bookstore/screenshots/qor_admin_add_book.png)
+
+![qor_admin_books1](https://raw.githubusercontent.com/qor/qor/docs_and_tutorial/example/tutorial/bookstore/screenshots/qor_admin_books1.png)
+
+
+#### Meta Module - Controlling display and editable fields in the admin
+
+Go to [http://localhost:9000/admin/books](http://localhost:9000/admin/books).
+Now comment the following line from [resources.go](https://github.com/qor/qor/blob/master/example/tutorial/bookstore/01/app/resources.go)
+
+	book.IndexAttrs("ID", "Title", "AuthorNames", "FormattedDate", "DisplayPrice")
+
+and reload the books admin page.
+
+![qor_admin_books2](https://raw.githubusercontent.com/qor/qor/docs_and_tutorial/example/tutorial/bookstore/screenshots/qor_admin_books2.png)
+
+You will see a much more crowded list: We had 5 attributes `"ID", "Title", "AuthorNames", "FormattedDate", and "DisplayPrice"`. `Id`, `Title`, and `ReleaseDate` are defined on our `Book` model, but the others are not. For the Authors field you only see a list of References to the `Author` objects - something like `[0xc208161bc0]`. We want the list of author names devided by `,` instead. You can achieve that by defining a `Meta` field:
+
+	book.Meta(&admin.Meta{
+		Name:  "AuthorNames",
+		Label: "Authors",
+		Valuer: func(value interface{}, context *qor.Context) interface{} {
+			if value == nil {
+				return value
+			}
+			book := value.(*Book)
+			if err := db.Model(&book).Related(&book.Authors, "Authors").Error; err != nil {
```

Comment:
This isn't correct:

```
 # github.com/qor/qor/example/tutorial/bookstore/01/app/resources
app/resources/resources.go:165: undefined: db
```

Oh, this code is already *present* in the config anyway...